doc/user/content/sql/system-catalog/mz_catalog.md

```diff
@@ -189,9 +189,11 @@ The `mz_egress_ips` table contains a row for each potential IP address that the
 system may connect to external systems from.
 
 <!-- RELATION_SPEC mz_catalog.mz_egress_ips -->
-Field       | Type     | Meaning
-------------|----------|--------
-`egress_ip` | [`text`] | The IP address.
+Field           | Type     | Meaning
+----------------|----------|--------
+`egress_ip`     | [`text`] | The start of the range of IP addresses.
+`prefix_length` | [`uint8`]   | The number of leading bits in the CIDR netmask.
```

Comment:
```suggestion
`prefix_length` | [`integer`] | The number of leading bits in the CIDR netmask.
```

Unsigned integers aren't well understood by SQL drivers; I think you're better off here using `integer` even though it's not quite as precise as a type.

Reply:
(Also if you're not already aware, `uint8` is a 64-bit integer, not an 8-byte integer.)